src/Microsoft.SqlTools.ServiceLayer/ShowPlan/Contracts/ExecutionPlanGraph.cs

```diff
@@ -0,0 +1,103 @@
+using System.Collections.Generic;
+
+namespace Microsoft.SqlTools.ServiceLayer.ShowPlan
+{
+    /// <summary>
+    /// Execution plan graph object that is sent over JSON RPC
+    /// </summary>
+    public class ExecutionPlanGraph
+    {
+        /// <summary>
+        /// Root of the execution plan tree
+        /// </summary>
+        public ExecutionPlanNode Root {get; set;}
```

Comment:
inconsistent spacing around the '{' and 'g' and ';' and '}' compared to other class.